packages/extension-chakra-storefront/jest-setup.js

```diff
@@ -137,6 +137,7 @@ jest.mock('@salesforce/cc-datacloud-typescript', () => ({
 // from nodejs util in testing environment.
 global.TextEncoder = require('util').TextEncoder
 global.TextDecoder = require('util').TextDecoder
+global.structuredClone = (v) => JSON.parse(JSON.stringify(v))
```

Comment:
I'm interested in knowing why we have to do this. From what I was able to dig up, that function should be available for the versions of node that we use. Is it the case that jest is somehow using an older version?

Reply:
I believe because we are using `jest-environment-jsdom-global` which does not include this function. Someone on this [thread](https://stackoverflow.com/questions/73607410/referenceerror-structuredclone-is-not-defined-using-jest-with-nodejs-typesc) stackoverflow mentioned that jsdom does not have this function in global object. 
![image](https://github.com/user-attachments/assets/f026f7aa-e6aa-4aae-bf20-ab3089bcce29)

Reply:
This fix is to address this error
![image](https://github.com/user-attachments/assets/4a9f506e-ffb1-47c3-85dc-7d4fc653e9fa)

Reply:
This is a perfect example to add code comment to explain the WHY 😉